### PR TITLE
aalhommada-node.js-week1

### PR DESCRIPTION
**Pull request by aalhommada**
i removed the first push by mistake , so i push a new branch a gain , with corrected codes

### CODE REVIEW

**Comment by aalhommada** [comment]
i check the code  hope it it working know

File: week1/homework/src/server.js [hunk 1]
```diff
@@ -2,16 +2,38 @@
...
+      state = 10;
+      response.writeHead(200, results);
+      response.write(JSON.stringify({ state: state }));
+    }
+ else {
+      response.writeHead(404, results);
+      response.write(JSON.stringify({ error: 'Not found' }));
+    }
 
+    response.end();
```
Comment (gajduk) [inline week1/homework/src/server.js]: response.end after all if-else statements :+1

File: week1/homework/src/server.js [hunk 2]
```diff
@@ -2,16 +2,38 @@
...
-function createServer(port) {
+function createServer() {
   let state = 10;
+  const results = { 'Content-Type': 'application/json' };
 
   const server = http.createServer((request, response) => {
-    // TODO: Write your homework code here
-  });
+    let url = request.url;
+    if (url === '/state') {
```
Comment (gajduk) [inline week1/homework/src/server.js]: use of tripple equals 👍

**Comment by aalhommada** [comment]
very nice , i will try to find some comparisons .

**Pull request closed without merging**